### PR TITLE
feat(tests): add unit tests for `isSameAddress` evm util

### PR DESCRIPTION
**Pull request by tcoratger**
This PR contains:
- Unit tests for `isSameAddress` EVM util function
- Unit test script to the CI

### CODE REVIEW

File: src/test/evm-utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    });
+
+    it("Should return true if both two addresses are egal (case insensitive)", async function () {
+      chai.expect(
+        isSameAddress("0x71fa225B8F9Aeb50B44F96743275837F8EB7694E", "0x71FA225B8F9Aeb50B44F96743275837F8EB7694E")
+      ).to.be.true;
+    });
+
+    it("Should return false if address A is null", async function () {
+      chai.expect(isSameAddress(null, "0x71FA225B8F9Aeb50B44F96743275837F8EB7694E")).to.be.false;
```
Comment (SAPikachu) [inline src/test/evm-utils.test.ts:24]:
```suggestion
      chai.expect(isSameAddress(null, "0x71fa225b8f9aeb50b44f96743275837f8eb7694e")).to.be.false;
```

File: src/test/evm-utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    });
+
+    it("Should return false if address B is null", async function () {
+      chai.expect(isSameAddress("0x71FA225B8F9Aeb50B44F96743275837F8EB7694E", null)).to.be.false;
```
Comment (SAPikachu) [inline src/test/evm-utils.test.ts:28]:
```suggestion
      chai.expect(isSameAddress("0x71fa225b8f9aeb50b44f96743275837f8eb7694e", null)).to.be.false;
```

File: src/test/evm-utils.test.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+/* eslint-disable no-unused-expressions */
+
+chai.use(chaiHttp);
+
+describe("EVM utils tests", async function () {
+  describe("isSameAddress (address A and address B)", async function () {
+    it("Should return true if both two addresses are egal (case sensitive)", async function () {
+      chai.expect(
+        isSameAddress("0x71FA225B8F9Aeb50B44F96743275837F8EB7694E", "0x71FA225B8F9Aeb50B44F96743275837F8EB7694E")
```
[the remark this answers is not in the document]
Comment (tcoratger) [reply src/test/evm-utils.test.ts:13]: @SAPikachu Agreed, more precise :)

**Review by SAPikachu — APPROVED** [review]
Looks good now. :)

**Pull request merged**